Basestation_Software.Web/Core/Services/RamanGraphService.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Basestation_Software.Web.Core.Services
+{
+	public class RamanGraphService
+	{
+        class DataItem
+        {
+            public int Raman_Shift { get; set; }
+            public double Intensity { get; set; }
+        }
```

Comment:
Remove this once DataItem class is declared in .Models project.

Reply:
Then you can use the DataItem from the .Models project by doing `using Basestation_Software.Models.RamanGraph` (or something similar)